tarantool/connection.py

```diff
@@ -1023,7 +1040,8 @@ def connect(self):
             if self.transport == SSL_TRANSPORT:
                 self.wrap_socket_ssl()
             self.handshake()
-            self.load_schema()
```

Comment:
It seems that my request to support explicit `update_schema` call is not so simple to implement (and it doesn't seem like a good design since its documented as a private API). Let's forget this one (since such use case is rather weird) and simply implement the following:
- initialize `self.schema` to `None`;
- if `fetch_schema` is `True`, build initial `self.schema` here, on `connect`, and load it;
- if `fetch_schema` is `False`, set `self.schema` to `None` here, on `connect` (just in case);
- if `self.schema` is `None`, raise errors on requests with schema;
- if `self.schema` is not `None`, reload the schema on receiving `SchemaReloadRequest`.

With this approach, `fetch_schema` meaning is "whether we fetch schema on connect or not", which is, I think, rather straightforward.

Let's remove explicit `update_schema`/`load_schema` examples from test. Since Python allows to mess with object internals, it still would be possible to turn schemaless connection to schemaful, but it shouldn't be our burden anymore.

Reply:
Maybe schema_version reset should also be processed

Reply:
Done

> Let's remove explicit update_schema/load_schema examples from test. Since Python allows to mess with object internals, it still would be possible to turn schemaless connection to schemaful, but it shouldn't be our burden anymore.

I don't quite understand what is meant, now `update_schema` is only used inside `test_08_schema_fetch_disable` when switching from schemaless to schemaful